### PR TITLE
Mechanicum Profile Fixes

### PR DESCRIPTION
**Pull request by stvnksslr**
> ⚠️  NOTE: use notes like this to emphasize something about the PR. This could
> include other PRs this PR is built on top of; reasons for why the PR is on hold; 
or anything else you would like > to draw attention to

## Description
* First issue: Magos prime, if you give him a melta gun, the weapon profile doesn't show up.
* the same goes for the magos dominus, so i think something's wonky with melta gun in particular
* the archmagos dominus still can't choose a specific power weapon. just "power weapon"
* the hardened armour on the abeyant is missing its description
apparently the rules for hardened armour can be found under the "breachers" entry in the legions book
* Ursarax are missing their lightning claws 
* Castellax don't have the atomantic shielding listed properly

### CODE REVIEW

**Comment by Mayegelt** [comment]
Looks like you have reversed the #1629 fix in LA

File: (HH) Legiones Astartes - Crusade Army List.cat [hunk 1]
```diff
@@ -49002,10 +49007,10 @@ If Castrmen Orth is your army’s Warlord, then all vehicles with the ‘Tank’
             <characteristic name="S" typeId="5323232344415441232323">7</characteristic>
             <characteristic name="Front" typeId="46726f6e7423232344415441232323">13</characteristic>
             <characteristic name="Side" typeId="5369646523232344415441232323">12</characteristic>
-            <characteristic name="Rear" typeId="5265617223232344415441232323">11</characteristic>
+            <characteristic name="Rear" typeId="5265617223232344415441232323">10</characteristic>
```
Comment (Mayegelt) [inline (HH) Legiones Astartes - Crusade Army List.cat]: Please change back to 11 as per #1629

File: (HH) Legiones Astartes - Crusade Army List.cat [hunk 2]
```diff
@@ -49002,10 +49007,10 @@ If Castrmen Orth is your army’s Warlord, then all vehicles with the ‘Tank’
...
             <characteristic name="I" typeId="4923232344415441232323">5</characteristic>
             <characteristic name="A" typeId="4123232344415441232323">3</characteristic>
-            <characteristic name="HP" typeId="485023232344415441232323">4</characteristic>
+            <characteristic name="HP" typeId="485023232344415441232323">3</characteristic>
```
Comment (Mayegelt) [inline (HH) Legiones Astartes - Crusade Army List.cat]: Please change back to 11 as per #1629

**Comment by Mayegelt** [comment]
Also looks like the Solar Aux changes made have not included the changes you had made from #1626

**Comment by capitaladot** [comment]
@stvnksslr thanks for your pull. It does look like you'll need to rebase.

File: (HH) Legiones Astartes - Crusade Army List.cat (hunk 2 above)
Comment (stvnksslr) [reply (HH) Legiones Astartes - Crusade Army List.cat]: will do not sure how that got flipped

**Comment by stvnksslr** [comment]
np yeah didnt realize it was desynch'd will fix.

**Pull request closed without merging**